File: src/main/java/problems/Uri1067_NumerosImpares.java
package problems;

import java.io.IOException;
import java.util.Scanner;

public class Uri1067_NumerosImpares {
    public static void main(String[] args) throws IOException {
        Scanner ler = new Scanner(System.in);
        int n = ler.nextInt();

        for (int j = 1; j <= n; j++) {
            if (j % 2 != 0) {
                System.out.println(j);
            }
        }
    }
}

File: src/main/kotlin/problems/Uri1067_NumerosImpares.kt
package problems

import java.util.Scanner

fun main(args: Array<String>) {
    val ler = Scanner(System.`in`)
    val n = ler.nextInt()
    for (j in 1..n step 2) {
        println(j)
    }
}

/*URI Online Judge | 1067
Números Ímpares
Adaptado por Neilor Tonin, URI  Brasil

Timelimit: 1
Leia um valor inteiro X (1 <= X <= 1000). Em seguida mostre os ímpares de 1 até X, um valor por linha, inclusive o X, se for o caso.

Entrada
O arquivo de entrada contém 1 valor inteiro qualquer.

Saída
Imprima todos os valores ímpares de 1 até X, inclusive X, se for o caso.

Exemplo de Entrada	Exemplo de Saída
8

1
3
5
7

 */